docs/guide.md

```diff
@@ -159,22 +159,29 @@ have a 5 minutes cache per default depending on the last access time of the file
 
 If you experience problems, consider clearing the cache via `ghcup gc --cache`.
 
-## Metadata
+## Release channels
 
-Metadata files are also called release or distribution channels. They describe tool versions, where to download them etc. and
-can be viewed here: [https://github.com/haskell/ghcup-metadata](https://github.com/haskell/ghcup-metadata).
+Release channels (also called distribution channels) are implemented by
+references to metadata files, which can be added using the
+`ghcup config add-release-channel` command.
```

Comment:
Perhaps an overview of both ways could be added here.

The URI of the metadata file can be specified to ghcup via configuration file, or for one-shot use via option `ghcup --url-source=<URI>`. The `ghcup config add-release-channel` command can be used for adding to configuration file.

Reply:
I'm happy to make these sorts of changes also and will do so.